README.md

```diff
@@ -7,87 +8,127 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![Twitter Follow](https://img.shields.io/twitter/follow/imagekitio?label=Follow&style=social)](https://twitter.com/ImagekitIo)
 
-ImageKit Python SDK allows you to use [image resizing](https://docs.imagekit.io/features/image-transformations), [optimization](https://docs.imagekit.io/features/image-optimization), [file uploading](https://docs.imagekit.io/api-reference/upload-file-api) and other [ImageKit APIs](https://docs.imagekit.io/api-reference/api-introduction) from applications written in the Python language.
+ImageKit Python SDK allows you to use [image resizing](https://docs.imagekit.io/features/image-transformations)
+, [optimization](https://docs.imagekit.io/features/image-optimization)
```

Comment:
Why these unnecessary new lines?

Reply:
We had verified that these don't break md file's preview version. The IDE was autoformatting this.